xla/service/gpu/ir_emitter_unnested.cc

```diff
@@ -2557,61 +2558,83 @@ static std::optional<GlobalDeviceId> DeviceConstraint(
   return std::nullopt;
 }
 
-absl::Status IrEmitterUnnested::EmitCopyStartThunk(
-    const HloCopyStartInstruction* instr) {
+absl::Status
+IrEmitterUnnested::EmitCopyStartThunk(const HloCopyStartInstruction *instr) {
```

Comment:
Let's separate formatting changes, would make the review easier.

Reply:
Done.